### ISSUE TITLE
vf-masthead short description

### ISSUE THREAD

**Issue by dbushell** [visual-framework/vf-wp#36]
In the theme's `partials/vs-masthead.php` include, the short description under the group name is from the Content Hub.

Todo:

* Use config instead of hard-coded API in template (`https://dev.beta.embl.org/api/v1/`)
* (idea) Move to EMBL Settings admin page with button to refresh cache
* (idea) store as standard WP blog description/tagline option?

**Comment by dbushell** [comment]
work in progress to complete after #27

**Comment by dbushell** [comment]
Merged into issue #27
